MolecularNodes/mda.py

```diff
@@ -0,0 +1,409 @@
+import bpy
+from bpy.app.handlers import persistent
+
+import MDAnalysis as mda
+import numpy as np
+import warnings
+import pickle
+import uuid
+import os
+
+from . import data
+from . import coll
+from . import obj
+from . import nodes
+
+
+class MDAnalysisSession:
+    # default location to store the session files
+    session_tmp_dir = f"{os.path.expanduser('~')}/.blender_mda_session/"
+
+    def __init__(
+        self,
+        universe,
+        name="atoms",
+        md_start=0,
+        md_end=-1,
+        md_step=1,
```

Comment:
We can probably get rid of the start / end / step. By linking the change of frames to the change of the frame inside of blender, the user can set the scene frame range to select frames, and it's probably best that it remains linked to the underlying universe, so we can just initialise the entire trajectory and you select the frames by scrubbing through the timeline.